src/libm-tester/iutsimdmain.c

```diff
@@ -0,0 +1,57 @@
+//          Copyright Naoki Shibata 2010 - 2017.
```

Comment:
Below is what I wrote in the commit log,

   iutsimd.c is now divided into two parts. This is needed to make it
   work with icc. In the previous implementation, the whole program is
   compiled with support for extended ISA (e.g. AVX512F), and this
   caused the IUT not to exit with the correct exit code when that
   ISA is not supported on the host.